src/main/native/ECKey.c

```diff
@@ -2121,12 +2121,14 @@ JNIEXPORT jbyteArray JNICALL Java_com_ibm_crypto_plus_provider_ock_NativeInterfa
   jbyteArray           retSecretBytes = NULL;
   size_t               secret_key_len = 0;
 
-  
-  if( debug ) gslogFunctionEntry(functionName);
+  if( debug ) {
+    gslogFunctionEntry(functionName);
+  }
 
   gen_ctx = ICC_EVP_PKEY_CTX_new(ockCtx,(ICC_EVP_PKEY *) ockPrivXecKey,NULL); /* Set private key */
-  if(gen_ctx == NULL) throwOCKException(env, 0, "NULL from ICC_EVP_PKEY_CTX_new"); 
-  else {
+  if ( NULL == gen_ctx ) {
```

Comment:
Can we fix the spaces in the parentheses too?

Reply:
Removed the spaces in the parentheses here and other locations in this method.